src/hook/usePersistedState.ts

```diff
@@ -6,14 +6,11 @@ const logger = createLogger('usePersistedState');
 
 export type PersistedStateKey = 'network' | 'accounts' | 'theme' | 'selected_push_topics';
 
-export function usePersistedState<T>(key: PersistedStateKey): [state: T | undefined, setState: (data: T) => void];
-export function usePersistedState<T>(key: PersistedStateKey, initialState: T): [state: T, setState: (data: T) => void];
-
 export function usePersistedState<T>(
   key: PersistedStateKey,
-  initialState?: T,
+  fallback?: T,
```

Comment:
I'm not sure I understand the changes here. We need the ability to set an initial state, this is not the same as always returning a fallback value since the initial state is only set once, the first time the hook runs. Also the fallback value will trigger an extra re-render when calling `setState(fallback)`, If one needs a fallback it would be better to do it in the component e.g:

```
const [value] = usePersistedState('key');
const x = Boolean(value) ? value : fallback;
```

Reply:
I understand what you mean, we would be optimizing for this specific use case. The idea of this hook is to have the same api (without lazy initialization) as `useState`.. The only issue is the async call, I hope we could solve it by using MMKV.